docs/framework/deployment/deployment-guide-for-developers.md

```diff
@@ -67,16 +58,14 @@ When you're ready to publish your app to a web server or other centralized locat
 
 ## Redistributable Packages
 
-The .NET Framework is available in two redistributable packages: web installer (bootstrapper) and offline installer (stand-alone redistributable). The following table compares the two packages.
+The .NET Framework is available in two redistributable packages: web installer (bootstrapper) and offline installer (stand-alone redistributable). All .NET Framework downloads are hosted on the [Download .NET Framework page](https://dotnet.microsoft.com/download/dotnet-framework/). The following table compares the two packages.
 
```

Comment:
Is the information here still true since now you seem to have only one installer link?

Reply:
Once you click on a specific framework link in that page, it is true.